### PR TITLE
koord-scheduler: add ElasticQuota Guaranteed usage

### PR DESCRIPTION
**Pull request by shaloulcy**
### Ⅰ. Describe what this PR does

add ElasticQuotaGuaranteeUsage featuregate go guarantee usage

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: apis/extension/elastic_quota.go [hunk 1]
```diff
@@ -45,6 +45,8 @@ const (
 	AnnotationResourceKeys    = QuotaKoordinatorPrefix + "/resource-keys"
 	AnnotationTotalResource   = QuotaKoordinatorPrefix + "/total-resource"
 	AnnotationQuotaNamespaces = QuotaKoordinatorPrefix + "/namespaces"
+	AnnotationGuarantee       = QuotaKoordinatorPrefix + "/guarantee"
```
Comment (eahydra) [inline apis/extension/elastic_quota.go:48]: guaranteed?

Comment (shaloulcy) [reply apis/extension/elastic_quota.go]: done

**Review by hormes — APPROVED** [review]

**Pull request merged**